ddp-workspace/projects/ddp-dsm-ui/src/app/clinical-page/clinical-page.component.html

```diff
@@ -0,0 +1,36 @@
+<div align="center" class="Width--100 Height--100">
+  <i class="fas fa-spinner fa-spin fa-5x Color--primary"></i>
+</div>
+<ng-container *ngIf="!loading">
+  <h1> Clinical Orders </h1>
+  <div align="right" style="width: 100%; height: 100%;">
```

Comment:
**#Optional**
I think it would be better, if we don't write styles inline, instead we can move them to css/scss file.